Sources/Rendering/Core/Picker/index.d.ts

```diff
@@ -86,13 +81,9 @@ export interface vtkPicker extends vtkAbstractPicker {
 	setMapperPosition(x: number, y: number, z: number): boolean;
 
 	/**
-	 * Set position in mapper coordinates of pick point.
-	 * @param {Vector3} mapperPosition The mapper coordinates of pick point.
-	 */
-	setMapperPositionFrom(mapperPosition: Vector3): boolean;
```

Comment:
This should not be removed, it's part of the methods
![Capture d’écran du 2024-01-13 10-41-22](https://github.com/Kitware/vtk-js/assets/878518/1661391b-3676-4f06-90c6-178c413a223f)